uelc/main/views.py

```diff
@@ -212,6 +206,7 @@ def notify_facilitators(self, request, path, notification):
         try:
             socket.recv(zmq.NOBLOCK)
         except zmq.ZMQError:
+            socket.recv()
```

Comment:
I am curious though what the root of the (ZMQ) error is . The main reason why I removed this notification was I thought the socket connections were running on top of each other. But these should be running concurrently, no?